workflows/gas_nomenclature.nf

```diff
@@ -72,29 +72,18 @@ workflow GAS_NOMENCLATURE {
     input = Channel.fromSamplesheet("input")
 
     // Ensure meta.id and mlst_file keys match; generate error report for samples where id ≠ key
-    id_key = INPUT_CHECK(input)
+    id_key = INPUT_ASSURE(input)
     ch_versions = ch_versions.mix(id_key.versions)
 
     // Update metadata to include the id_key.match data
     match = id_key.match.map { meta, file, json ->
         def id_match = file.text.trim()
         [meta + [id_match: id_match == 'True'], json]
     }
-
-    // If samples have a disparity between meta.id and JSON key: Exclude the queried samples OR halt the pipeline with an error if sample has an associated cluster address (reference)
-    new_input = match.filter { meta, json ->
-        if (meta.id_match) {
-            return true // Keep the sample
-        } else if (meta.address == null && !meta.id_match) {
-            return false // Remove the sample
-        } else if (meta.address != null && !meta.id_match) {
-            // Exit with error statement
-            throw new RuntimeException("Pipeline exiting: sample with ID ${meta.id} does not have matching MLST JSON file.")
-        }
-    }
+    match.view()
```

Comment:
Do we want to leave the `.view()` in or was this for debugging?

Reply:
Haha - we don't. 
Although highlighting this also made me realize that we no longer need to store the `id_match` boolean in the `meta` data as we won't be removing any samples from the analysis.

I have updated the python code, the workflow, and the `input_assure` process to simplify this here: [95e40f6](https://github.com/phac-nml/gasnomenclature/pull/13/commits/95e40f6979ab60cbe6f8531bb72996c45cf1c56f)